graphics/grafx2/Portfile

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                grafx2
+version             2.5
+categories          graphics
+maintainers         @miniupnp
+platforms           darwin
+license             GPL-2
+
+description         The ultimate 256-color painting program
+long_description    GrafX2 is a bitmap paint program inspired by the Amiga programs \
+                    Deluxe Paint and Brilliance. Specialized in 256-color drawing, \
+                    it includes a very large number of tools and effects that make \
+                    it particularly suitable for pixel art, game graphics, and generally \
+                    any detailed graphics painted with a mouse.
+
+homepage            http://grafx2.chez.com/
+distname            ${name}-v${version}
+master_sites        https://gitlab.com/GrafX2/grafX2/-/archive/v${version}/::gfx2 \
+                    sourceforge:project/recoil/recoil/4.2.0:recoil
+distfiles           ${distname}.tar.bz2:gfx2 \
+                    recoil-4.2.0.tar.gz:recoil
+use_bzip2           yes
+checksums           ${distname}.tar.bz2 \
+                    sha256 cf634e0f1d94bc5131a4de4490d4bf07e2965d774687add2487a9fa0240aeb10 \
+                    rmd160 692e8c871ade535db741bfff730cb74bc3398327 \
```

Comment:
You should probably have "size" for both of your distfiles.

Reply:
done